web/packages/extension/assets/options.html

```diff
@@ -58,6 +58,13 @@
                 </select>
                 <label for="preferred_renderer">Preferred renderer</label>
             </div>
+            <div class="option select">
+                <select id="device_font_renderer">
+                    <option value="embedded">Embedded</option>
```

Comment:
For i18n, should we add id `id="device_font_renderer_embedded" and translation `settings_device_font_renderer_embedded` (and canvas)?